google_compute_engine/accounts/accounts_utils.py

```diff
@@ -76,10 +76,16 @@ def _CreateSudoersGroup(self):
         self.logger.warning('Could not create the sudoers group. %s.', str(e))
 
     if not os.path.exists(self.google_sudoers_file):
-      with open(self.google_sudoers_file, 'w') as group:
-        message = '%{0} ALL=(ALL:ALL) NOPASSWD:ALL'.format(
-            self.google_sudoers_group)
-        group.write(message)
+      try:
+        with open(self.google_sudoers_file, 'w') as group:
+          message = '%{0} ALL=(ALL:ALL) NOPASSWD:ALL'.format(
+              self.google_sudoers_group)
+          group.write(message)
+      except IOError as e:
+        self.logger.error(
+            'Could not write sudoers file. %s\n%s' % (
+                self.google_sudoers_file, str(e)))
+        return None
```

Comment:
This function doesn't have a return type. Instead, you can add an `else:` that will then set permissions on the file.